roofit/roofit/src/RooFitComputeLib.cxx

```diff
@@ -11,6 +11,13 @@
 using namespace BatchHelpers;
 
 namespace RooFitCompute {
+  /**
+   * \brief Contains the part of the code of the RooFitCompute Library that needs to be compiled for every different cpu architecture.
+   *
+   * RF_ARCH is a macro that is defined by cmake. The macro gets a different name for each copy of the library, namely SSE4, AVX, AVX2, AVX512.
```

Comment:
`AVX512F`, I assume. You can weaken the statement by adding an `e.g.` instead of `namely`.

Reply:
Outdated. Comment is from second-to-last commit, and gets invalidated by last commit.